### ISSUE TITLE
Consider switching to flutter_timezone

### ISSUE THREAD

**Issue by tjarvstrand** [builttoroam/device_calendar#495]
flutter_native_timezone has been abandoned for more than 2 years. 

I've created a fork, [flutter_timezone](https://pub.dev/packages/flutter_timezone) that you might consider switching to instead as some [users are starting to go get build issues](https://github.com/pinkfish/flutter_native_timezone/pull/58) with the old dependency.

**Comment by techwnn** [comment]
@tjarvstrand , I checked the latest code, the deps has been deleted.
I think we need it published to pub.dev with a patch fix like 4.3.2?

**Comment by davehorner** [comment]
I have no opinion of matter.  I took on the dependency and it broke my build.  Just trying to help with a patch for updates - as they can build with minor tweaks.  If the forked dep should be used cool, removing it is fine too just wonder what we lost in taking it out.

**Comment by ihadabs** [comment]
Any update?

**Comment by IVLIVS-III** [comment]
Sadly all current and past maintainers have next to no time left. Tbh most of us became maintainers to fix a bug.

To get this issue fixed as soon as possible, it would be best if someone provides a pull request that is ready to be merged.

**Comment by tjarvstrand** [comment]
No changes are needed, only a release to get the changes out that are already in `main`

**Comment by behnamsattar** [comment]
@tjarvstrand I don't see the fork [flutter_timezone](https://pub.dev/packages/flutter_timezone) being used on `master`. To be clear, don't we need a PR replacing `flutter_native_timezone` with `flutter_timezone`?

**Comment by behnamsattar** [comment]
The example project is still relying on `flutter_native_timezone` and it's included in the dependencies as a result.

**Comment by tjarvstrand** [comment]
> The example project is still relying on flutter_native_timezone and it's included in the dependencies as a result.

No, that shouldn't affect the dependencies of the released package. 

The last release was 4.3.1, made 5 months ago, apparently from `master`. There seems to have been some cherry-picking going on from `develop` to `master` but notably, [this commit](https://github.com/builttoroam/device_calendar/commit/aa31c46cd694cf468dbb983bb9bbc9d75315cd50) does not seem to have been included, which is why the 4.3.1 still has the `flutter_native_timezone` dependency.

**Comment by tjarvstrand** [comment]
> As a workaround you can use the following snippet in your pubspec.yaml

That's great. I think there's a risk however, of other people hitting the same issue and the probability that they will stumble upon this thread isn't super high. It could be worth making a bump to 4.3.2 (or even 4.3.1+1 with what's actually in the tagged commit) to avoid that scenario.

Just my 2 cents though, as I'm not really affected by this :)

**Comment by galacticgibbon** [comment]
My package doesn't have this issue [device_calendar_plus](https://pub.dev/packages/device_calendar_plus)

**Comment by tjarvstrand** [comment]
@galacticgibbon This issue was resolved three years ago, with the release of 4.3.2

**Comment by galacticgibbon** [comment]
@tjarvstrand The project is not maintained so it's helpful for people to know alternatives exist
